raptorWeb/authprofiles/views.py

```diff
@@ -36,7 +36,6 @@ def get(self, request):
             dictionary['user_path'] = BASE_USER_URL
```

Comment:
Seems like all instances of `dictionary['user_path'] = BASE_USER_URL` in this `views.py` can be removed as that info is added to context by the context processor.